plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/BuiltinCostModel.hs

```diff
@@ -21,6 +21,7 @@ module PlutusCore.Evaluation.Machine.BuiltinCostModel
     , ModelConstantOrLinear(..)
     , ModelConstantOrTwoArguments(..)
     , ModelLinearSize(..)
+    , ModelLinearSizeTwoVariables(..)
```

Comment:
I wondered whether to call this `ModelLinearSizeTwoArguments` but then came down on the side of `Variables` instead because I thought that "arguments" referred to the arguments of the builtin and "variables` referred to the aruguments of the costing function.  I may change my mind about that.

Reply:
/benchmark validation

Reply:
I've now changed the names again and added a big comment [here](https://github.com/input-output-hk/plutus/blob/3ef1b15f7c65cb87ffc6e134ef627a6168ff10d3/plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/CostingFun/Core.hs#L195). As mentioned in some of the other comments that I made on the PR, some renaming/refactoring might make this stuff a bit clearer, but this isn't the place to do it.